### ISSUE TITLE
unknown_types, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1333]
https://github.com/sapmentors/abap2xlsx/blob/master/src/zcl_excel_autofilter.clas.abap

```
./src/zcl_excel_autofilter.clas.abap[266, 18] - Type of "<ls_sheet_content>" contains unknown: Type error, not a table type me->worksheet->sheet_content (unknown_types)
./src/zcl_excel_autofilter.clas.abap[307, 18] - Type of "<ls_sheet_content>" contains unknown: Type error, not a table type me->worksheet->sheet_content (unknown_types)
```

**Comment by larshp** [comment]
reproduced as

```abap
CLASS foobar DEFINITION.
  PUBLIC SECTION.
    DATA worksheet TYPE REF TO foobar.
    DATA sheet_content TYPE STANDARD TABLE OF i WITH DEFAULT KEY.
    METHODS moo.
ENDCLASS.

CLASS foobar IMPLEMENTATION.
  METHOD moo.
    FIELD-SYMBOLS: <bar> LIKE LINE OF me->worksheet->sheet_content.
    WRITE <bar>.
  ENDMETHOD.
ENDCLASS.
```
